kamon-akka-2.4.x/src/main/scala/kamon/akka/instrumentation/ActorMonitor.scala

```diff
@@ -125,9 +163,21 @@ object ActorMonitors {
     }
 
     def processFailure(failure: Throwable): Unit = {}
+    def processDroppedMessage(count: Long): Unit = {}
     def cleanup(): Unit = {
       Metrics.forSystem(cellInfo.systemName).activeActors.decrement()
     }
+
+    def processMessageStart(envelopeContext: TimestampedContext, envelope: Envelope): Traveler = {
+      import CloseableSyntax._
+
+      processedMessagesCounter.increment()
+      val scope = Kamon.storeContext(envelopeContext.context)
+      Traveler(envelopeContext, scope.toCloseable)
```

Comment:
Similar to the `Traveler` situation.. why allocate a `Closeable` here if we don't really need it?